### DISCUSSION TITLE
unable to get on-premise data to display

### DISCUSSION THREAD

**Discussion by Sommie99** [cloud-carbon-footprint/cloud-carbon-footprint#1369] [General]
Following the standard process, I have managed to connect the AWS information, however when attempting to use the on-premise csv and running the 'yarn run estimate-on-premise-data --onPremiseInput on_premise_data_input.csv' i generate the estimations csv.

Following a yarn start, that charts still only show information from AWS.  Have i forgotten to update anything?

I have tried using the test data to ensure i didnt enter the information wrongly and still the same.  Any ideas?

**Comment by camcash17** [comment]
Hi @Sommie99, would it be possible to create a new issue for this and also provide detailed information about your configuration to help debug?

Thanks!

**Reply by Sommie99** [reply]
yes, no problem.  i thought it may have been a me issue rather than a system issue
